<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWindow</class>
 <widget class="QDialog" name="StatisticsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/window/film-statistic</normaloff>:/window/film-statistic</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMainStatistics">
      <attribute name="title">
       <string>Main statistics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="labelTotalFilmsInLibrary">
             <property name="text">
              <string>Total films in the library:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelFilmsViewed">
             <property name="text">
              <string>Films viewed:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelTotalViews">
             <property name="text">
              <string>Total views:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelWastedTime">
             <property name="text">
              <string>Wasted time:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="lTotalFilmsInLibrary">
             <property name="text">
              <string notr="true">0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lFilmsViewed">
             <property name="text">
              <string notr="true">0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lTotalViews">
             <property name="text">
              <string notr="true">0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lWastedTime">
             <property name="text">
              <string notr="true">0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelSpace1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMostPopularFilms">
         <property name="text">
          <string>Most popular films:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lwMostPopularFilms">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabReset">
      <attribute name="title">
       <string>Reset</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelNonono">
         <property name="text">
          <string>Warning! This action will reset the counter of views and can not be undone.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="hs1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bReset">
           <property name="styleSheet">
            <string notr="true">color: rgb(200, 0, 0);</string>
           </property>
           <property name="text">
            <string>Reset statistics</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hs2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StatisticsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
